relay-server/src/services/processor/event.rs

```diff
@@ -420,6 +420,26 @@ pub fn has_unprintable_fields(event: &Annotated<Event>) -> bool {
     }
 }
 
+//TODO: comment
+//TODO: add test: all incr/no incr cases, envelope unchanged
+pub fn emit_feedback_metrics(envelope: &Envelope) -> i64 {
```

Comment:
Why does this function return the count? To make it testable?

Reply:
Removed the return value since we're ok with no tests